packages/create-remix/templates/arc/README.md

```diff
@@ -2,37 +2,40 @@
 
 - [Remix Docs](https://remix.run/docs)
 
-## Architect Setup
 
-When deploying to AWS Lambda with Architect, you'll need:
+## Development
 
-- Architect (`arc`) CLI
-- AWS SDK
+Both Remix and Architect have their own local servers, each of which is responsible for unique tasks related to local development.
 
-Architect recommends installing these globally:
+To start both alongside one another, run:
 
 ```sh
-$ npm i -g @architect/architect aws-sdk
+$ npm run dev
 ```
 
-## Development
+Open up [http://localhost:3333](http://localhost:3333) and you should be ready to go!
+
 
-You will be running two processes during development when using Architect as your server.
+### TypeScript
```

Comment:
Does all this make sense in the context of Remix? I know a lot of Remix + Arc folks have been trying to use TS, but have been a bit confused as to where to start, so I was thinking this might help shortcut them. Up to y'all!

Reply:
unfortunately using `@architect/plugin-typescript` for all lambdas causes a "lambda not found error" for the remix app, perhaps we can mention the config.arc approach instead? 

![](https://user-images.githubusercontent.com/11698668/157116598-7d22d931-e74d-4c00-ae87-e446189964b2.png)

![](https://user-images.githubusercontent.com/11698668/157116592-10d39378-67bd-4974-a3c8-8b08e2e521b7.png)

Reply:
lol wut, today it works

Reply:
i think instead of mentioning the installation of the typescript plugin directly, we can add a `arc-ts` directory and add the dependency and full app.arc there